zk-token-sdk/src/instruction/close_account.rs

```diff
@@ -109,14 +109,14 @@ mod test {
     use {
         super::*,
         crate::encryption::{
-            elgamal::ElGamalKeypair,
-            pedersen::{Pedersen, PedersenDecryptHandle, PedersenOpening},
+            elgamal::{DecryptHandle, ElGamalKeypair},
+            pedersen::{Pedersen, PedersenOpening},
         },
     };
 
     #[test]
     fn test_close_account_correctness() {
-        let source_keypair = ElGamalKeypair::default();
+        let source_keypair = ElGamalKeypair::random();
```

Comment:
nit: I'd prefer using the name `ElGamalKeypair::new_rand()` for consistency with other solana-sdk structs (`Pubkey`, `Hash`)

Reply:
Yep, sounds good.